src/editors/textEditor.js

```diff
@@ -381,12 +381,12 @@ class TextEditor extends BaseEditor {
         break;
     }
 
-    if (colHeadersCount && this.hot.getSelectedLast()[0] === 0 ||
+    if (colHeadersCount && this.hot.getSelectedLast()[0] <= 0 ||
         (settings.fixedRowsBottom && this.hot.getSelectedLast()[0] === totalRowsCount - settings.fixedRowsBottom)) {
```

Comment:
Shouldn't it be:

```suggestion
        (settings.fixedRowsBottom && this.hot.getSelectedLast()[0] <= totalRowsCount - settings.fixedRowsBottom)) {
```

right now?